### DISCUSSION TITLE
When extracting the .rar "Access is denied" 7zip

### DISCUSSION THREAD

**Discussion by TheScentOfYourShampooOnTheFlowers** [ALTaleX531/TranslucentFlyouts#96] [Show and tell]
![image](https://github.com/ALTaleX531/TranslucentFlyouts/assets/143460577/f447525b-cc45-43e9-acf8-114f76f24d8d)

**Comment by ALTaleX531** [comment]
https://github.com/ALTaleX531/TranslucentFlyouts/issues/60#issuecomment-1701475057
